site/sr-Latn.njk

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+meta_title: Pokret za zaustavljanje COVID-19 pandemije
+meta_description: Pridonesi borbi i spasi živote ne radeći nista! Jednostavno ostani u jebenoj kući!
+body_class: home
+title: '#StayTheFuckHome'
+lang: sr-Latn
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">Pokret za zaustavljanje COVID-19 pandemije</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Odgovorne agencije i medicinsko osoblje daju sve od sebe kako bi stavili pod kontrolu širenje 
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
+                 virusa i ograničili <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 pandemiju.</a>
+				Nije vreme za sebičnost, i dužnost je svakog građanina na celom planetu da pridonese ovoj borbi.
+                <br /><br />
+                Ovog puta stvarno ne treba napraviti puno da bi potencijalno spasio nečiji zivot, i reći ću ovo jednostavno tako da svatko može da razume: <strong>Jednostavno ostani u jebenoj kući!</strong>
+            </p>
+
+            <h2 class="text-3xl">Manifesto kućne karantene (samoizolacije)</h2>
+
+            <p>
+                Bez dobro testiranih tretmana lečenja i verovatnosti da dobijemo delotvorno cepivo jos bar godinu dana, jedini delotvoran način 
+                da se ova pandemija obuzda je dati ovom virusu manje šanse za širenje. Sledeća lista je najučinkovitiji način za borbu protiv pandemije, i ove jednostavne mjere stvarno mogu pomoći da promenimo stvari.
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">Bez panike!</li>
+                <li class="pb-2">Redovito perite ruke. Ako kašljete ili kišete, radite to u lakat ili u maramicu.</li>
+                <li class="pb-2">Pokušajte dirati vlastito lice što je manje moguće, pogotovo oči, usta i nos.</li>
+                <li class="pb-2">Prakticirajte socijalno udaljavanje bez zagrljaja, poljubaca, rukovanja.</li>
```

Comment:
"Prakticirajte socijalno udaljavanje" sounds like Google Translate at work. A less literal translation ("ograničite socijalne kontakte" or something) would probably be better.